/**:
  ros__parameters:
    debug: false
    vehicle_ip: "tcp:navio.local:5678"
    mission_filename: "MisionesLoyola_dron_2.kml"
    vehicle_id: 1
    mqtt_addr: 127.0.0.1

dronekit_node:
  ros__parameters:

    timeout: 15

main_node:
  ros__parameters:


mqtt_node:
  ros__parameters:



sensor_module_node:
  ros__parameters:
    database_name: "LOCAL_DATABASE.db"
    USB_string: "USBPort1"
    timeout: 6
    baudrate: 115200
    pump_parameters:
      charging_time: 7
      discharging_time: 2
      serial_string: "/dev/ttyACM0"
      mode: "BuiltInPin"